autoload/ale/rename.vim

```diff
@@ -116,7 +116,7 @@ function! ale#rename#HandleLSPResponse(conn_id, response) abort
         \       'changes': l:changes,
         \   },
         \   {
-        \       'should_save': 1,
```

Comment:
Actually seems this was the behavior before?

Reply:
Yeah it was, the reason why I changed the default is in the description of the first commit.

Reply:
Done, also added an option to disable this behavior.